### PR TITLE
Added Jupyter SwePolice and Wikidata

### PR DESCRIPTION
**Pull request by salgo60**
SBK the links in Readme.mb  feels wrong

### CODE REVIEW

**Comment by salgo60** [comment]
Had some problems with  ipynb and they didnt load....

File: Wikidata/Wikidata Swedish Primeministers.ipynb [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from IPython.display import HTML\n",
```
Comment (consideRatio) [inline Wikidata/Wikidata Swedish Primeministers.ipynb:48]: There is also the YouTubeVideo class that can help out efficiently to accomplish the same thing!

**Review by consideRatio — APPROVED** [review]

**Comment by consideRatio** [comment]
Thank you @salgo60! I'm very happy that you have provided some initial examples here! I'll make it easy to access this repo from the JupyterHub during the hackaton!

File: README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# hack-for-sweden Examples
+## Swedish Police events
+See [SWEPoliceevents_H4S.ipynb](SWEPoliceEvents/SWEPoliceevents_H4S.ipynb)
```
Comment (consideRatio) [inline README.md:3]: The link is broken due to case sensitivity

**Pull request merged**